src/wolff_producer.erl

```diff
@@ -199,11 +199,15 @@ do_init(#{client_id := ClientId,
           partition := Partition,
           config := Config0
          } = St) ->
+  AliasPathSegment = case maps:find(alias, Config0) of
```

Comment:
alias should be added to `config_key` and `config` type spec

Reply:
https://github.com/kafka4beam/wolff/blob/45ad5df2282ebfbc116034027dbce25be8e96d89/src/wolff_producers.erl#L49

Reply:
i mean config_key() type spec in THIS module (`wolff_producer`).
and there should be no need to do `alias | wolff_producer:config_key()` in `wolff_producers`